runtime/binding-kafka/src/test/java/io/aklivity/zilla/runtime/binding/kafka/internal/stream/CacheMergedIT.java

```diff
@@ -185,13 +185,12 @@ public void shouldFetchMergedMessagesWithNoFilter() throws Exception
         k3po.finish();
     }
 
-    @Ignore("GitHub Actions")
     @Test
     @Configuration("cache.options.merged.yaml")
     @Specification({
         "${app}/merged.fetch.filter.sync/client",
         "${app}/unmerged.fetch.filter.sync/server"})
-    public void shouldFetchMergedMessagesLatest() throws Exception
+    public void shouldFetchMergedLatestMessages() throws Exception
```

Comment:
Usually we name the test methods according to the spec scenarios.
In this case, i think that would be `shouldFetchMergedFilterSync`, works?